vault/token_store.go

```diff
@@ -1818,10 +1818,11 @@ func (ts *TokenStore) handleTidy(ctx context.Context, req *logical.Request, data
 				}
 
 				var deletedChildrenCount int64
-				for _, child := range children {
+				for index, child := range children {
 					countParentList++
 					if countParentList%500 == 0 {
-						ts.logger.Info("checking validity of tokens in secondary index list", "progress", countParentList)
+						percentComplete := float64(index)/float64(len(children))*100
```

Comment:
This should either be `index+1` or use `countParentList` (same for the other ones).

Reply:
Thanks! I made a typo when changing it.